src/content/developers/docs/accounts/index.md

```diff
@@ -67,7 +67,7 @@ Example:
 
 `fffffffffffffffffffffffffffffffebaaedce6af48a03bbfd25e8cd036415f`
 
-The public key is generated from the private key using the [Elliptic Curve Digital Signature Algorithm](https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm). You get a public address for your account by taking the last 20 bytes of the SHA3 hash of the private key and adding `0x` to the beginning.
+The public key is generated from the private key using the [Elliptic Curve Digital Signature Algorithm](https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm). You get a public address for your account by taking the last 20 bytes of the hash of the public key and adding `0x` to the beginning.
```

Comment:
@Rub3cula  🤦🏻  Good call, this was my bad. Read it too quickly while reviewing the last deploy.

Did you remove SHA3 for a reason? I would leave it in personally, either that or Keccak-256

Reply:
Very understandable!
I agree, Keccak-256 would be more correct I think